packages/docs/src/pages/qwikcity/content/component.mdx

```diff
@@ -1,3 +1,76 @@
 ---
-title: TODO
+title: Qwikcity - Components
 ---
+
+Components is a way of creating content using Qwik components. This is simply a file that contains a component that is exported as `default` export.
+
+Let's assume you have the router set up such as this:
+```
+- src/
+  - routes/
+    - some/           
+      - path/
+        - index.tsx              # http://server/some/path
+```
+
+File: `src/routes/some/path/index.tsx`:
```

Comment:
I collected some single sentences and brought them together.
In my view, it is easier to read. A paragraph also stands out a bit more in comparison to a single line between code-blocks.

```suggestion
Now, inside `index.tsx`you can create a component is exported as `default` export.
Content is by writing a template with [JSX](https://reactjs.org/docs/introducing-jsx.html)-Syntax.
Every component-Function returns the template that should be rendered on the screen.
If you want to learn more about the component anathomy, please chekout our [articles about Components](https://qwik.builder.io/docs/components/overview).

The following component will be rendered at `http://server/some/path` according to the directory-structure, we have set up above. 
```

Reply:
Incorporated.